specification/evaluation-context/evaluation-context.md

```diff
@@ -1,5 +1,42 @@
 # Evaluation Context
 
+**Status**: [Experimental](../README.md#document-statuses)
+
 ## Overview
 
-// TODO: this is a stub.
+The `evaluation context` provides ambient information for the purposes of flag evaluation. Contextual data may be used as the basis for targeting, including rule-based evaluation, overrides for specific subjects, or fractional flag evaluation.
+
+### Fields
+
+##### Requirement 3.1
```

Comment:
Why start at 3.1?

Reply:
Oh - I had each md file start at a different number. So Section 1 was evaluation API, second 2 was providers, 3 is evaluation context, etc. 

I see now that `hooks` starts at 1, so we werent on the same page here. We should review this.

Reply:
Seems like we should also add it to the linter. :)